src/agent/agent.c

```diff
@@ -1011,6 +1070,103 @@ static int agent_method_unsubscribe(sd_bus_message *m, void *userdata, UNUSED sd
         return sd_bus_reply_method_return(m, "");
 }
 
+/*************************************************************************
+ ********** org.containers.hirte.internal.Agent.StartDep *****************
+ ************************************************************************/
+
+static char *get_dep_unit(const char *unit_name) {
+        return strcat_dup("hirte-dep@", unit_name);
```

Comment:
Could we move `hirte-dep@` to a constant, e.g. in protocol.h (although I am not sure if it makes sense to move it there)?

Reply:
I dunno, its already abstracted away into a single function. Not sure having it in the header helps.